Assets/MRTK/Examples/Demos/ScrollingObjectCollection/Prefabs/ScrollingOC_PressableBtn_32x96.prefab

```diff
@@ -16110,17 +17174,19 @@ MeshRenderer:
   m_MotionVectors: 1
   m_LightProbeUsage: 1
   m_ReflectionProbeUsage: 1
+  m_RayTracingMode: 2
   m_RenderingLayerMask: 4294967295
   m_RendererPriority: 0
   m_Materials:
-  - {fileID: 2100000, guid: fa419ab56051229449e3b813df8f295f, type: 2}
+  - {fileID: 0}
```

Comment:
Does this material need to be re-added? Or is it okay without?

Reply:
There are a couple of these throughout the file

Reply:
I'll go through this again (this was fixing an issue with the button not disappearing when it went beyond the clipping bounds)